packages/cli/src/lib/commands/topic.ts

```diff
@@ -14,15 +14,14 @@ export const topic: CommandDefinition = (program) => {
 
     topicCmd.command("send <topic> [<file>]")
         .option("-t, --content-type <value>", "Content-Type", "text/plain")
-        .option("-e, --end", "x-end-stream", false)
         .description("send data to topic")
-        .action(async (topicName, filename, { contentType, end }) => displayEntity(
```

Comment:
@pietrzakacper I wouldn't remove this feature and I would actually add another command to explicitly `end` topic, called... uhm... `end`... ;)

It's ok to add it in another PR.